packages/cli/src/controllers/users.controller.ts

```diff
@@ -329,73 +327,47 @@ export class UsersController {
 
 	@Patch('/:id/role')
 	@RequireGlobalScope('user:changeRole')
-	async changeRole(req: UserRequest.ChangeRole) {
+	async changeGlobalRole(req: UserRequest.ChangeRole) {
 		const {
-			MISSING_NEW_ROLE_KEY,
-			MISSING_NEW_ROLE_VALUE,
+			INVALID_PAYLOAD,
 			NO_ADMIN_ON_OWNER,
-			NO_USER_TO_OWNER,
 			NO_USER,
 			NO_OWNER_ON_OWNER,
 			NO_ADMIN_IF_UNLICENSED,
 		} = UsersController.ERROR_MESSAGES.CHANGE_ROLE;
 
-		const { newRole } = req.body;
-
-		if (!newRole) {
-			throw new BadRequestError(MISSING_NEW_ROLE_KEY);
-		}
-
-		if (!newRole.name || !newRole.scope) {
-			throw new BadRequestError(MISSING_NEW_ROLE_VALUE);
+		const { roleName: newRole } = req.body;
+		if (!newRole || !['member', 'admin'].includes(newRole)) {
+			throw new BadRequestError(INVALID_PAYLOAD);
 		}
 
-		if (newRole.scope === 'global' && newRole.name === 'owner') {
-			throw new UnauthorizedError(NO_USER_TO_OWNER);
+		if (!this.license.isAdvancedPermissionsLicensed()) {
```

Comment:
Now that this check is standalone, should we use `Licensed()` ? Also `NO_ADMIN_IF_UNLICENSED` is not really accurately named anymore as _any_ action here is off limits if unlicensed.